src/machine.tex

```diff
@@ -1830,6 +1830,54 @@ \subsection{Machine Bad Bits ({\tt mtval}) Register}
 
 \section{Machine-Mode Privileged Instructions}
 
+\subsection{Environment Call and Breakpoint}
+
+\vspace{-0.2in}
+\begin{center}
+\begin{tabular}{M@{}R@{}F@{}R@{}S}
+\\
+\instbitrange{31}{20} &
+\instbitrange{19}{15} &
+\instbitrange{14}{12} &
+\instbitrange{11}{7} &
+\instbitrange{6}{0} \\
+\hline
+\multicolumn{1}{|c|}{funct12} &
+\multicolumn{1}{c|}{rs1} &
+\multicolumn{1}{c|}{funct3} &
+\multicolumn{1}{c|}{rd} &
+\multicolumn{1}{c|}{opcode} \\
+\hline
+12 & 5 & 3 & 5 & 7 \\
+ECALL   & 0 & PRIV & 0 & SYSTEM \\
+EBREAK  & 0 & PRIV & 0 & SYSTEM \\
+\end{tabular}
+\end{center}
+
+The ECALL instruction is used to make a request to the supporting execution
+environment.  When executed in U-mode, S-mode, or M-mode, it generates an
```

Comment:
The user spec already implies the runtime behavior is defined elsewhere

> The ABI for the system will define how parameters for the environment request...

and indeed I think it's best to leave it described as an ABI thing.